compiler/src/dmd/dtemplate.d

```diff
@@ -2691,7 +1731,7 @@ private size_t templateIdentifierLookup(Identifier id, TemplateParameters* param
     return IDX_NOTFOUND;
 }
 
-private size_t templateParameterLookup(Type tparam, TemplateParameters* parameters)
```

Comment:
maybe this is a indication that this one and the following functions should be moved along with `deduceFunctionTemplateMatch`?

Reply:
This is an idea for a follow up PR.

Reply:
You're right, but this PR is already a thousand lines!